app/csv.go

```diff
@@ -0,0 +1,25 @@
+package app
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+)
+
+func CSVReader() [][]string {
```

Comment:
This function name doesn't really tell me what it's supposed to do, it could probably be better located on a dedicated package to handle anything that has to do with reading the CSV data.

I would suggest not only reading and returning the rows, but also converting the data that is in the csv and returning an array of Pokemons, then in the usecase package you could check for the specific ID.

So this function could be in a package called `csvdata` or something like that, and the function could be named `GetPokemons`